adapters/sweden.js

```diff
@@ -0,0 +1,174 @@
+      case 'Uppsala Marsta':
+        return {'latitude': 59.92596545, 'longitude': 17.58700716};
+      case 'E4':
+        return {'latitude': 59.48583073, 'longitude': 17.91964065};
+      case 'Hornsgatan':
+        return {'latitude': 59.31713214, 'longitude': 18.04878744};
+      case 'Sveavägen':
+        return {'latitude': 59.34516113, 'longitude': 18.05428175};
+      case 'Folkungagatan':
+        return {'latitude': 59.31462368, 'longitude': 18.07585555};
+      case 'Botkyrka, Hågelbyleden':
+        return {'latitude': 59.23705806, 'longitude': 17.83833241};
+      case 'Gröndalsskolan':
+        return {'latitude': 59.31349142, 'longitude': 18.00469473};
+      case 'Fleminggatan (projekt)':
+        return {'latitude': 59.33375997, 'longitude': 18.03684915};
+      case 'E4 Häggvik':
+        return {'latitude': 59.44353901, 'longitude': 17.92236122};
+      case 'Norrlandsgatan':
+        return {'latitude': 59.33635627, 'longitude': 18.07062632};
+      case 'Södertalje, Turingegatan':
+        return {'latitude': 59.19812352, 'longitude': 17.62108719};
+      case 'Ekmansväg':
+        return {'latitude': 59.48900019, 'longitude': 17.92020954};
+      case 'Eriksbergsskolan':
+        return {'latitude': 59.41018492, 'longitude': 17.95779851};
+      case 'Falun, Svärdsjögatan':
+        return {'latitude': 60.60798503, 'longitude': 15.63367903};
+      case 'Södertalje, Birkakorset':
+        return {'latitude': 59.20135294, 'longitude': 17.63475503};
+      case 'Lilla Essingen (E4/E20)':
+        return {'latitude': 59.32551867, 'longitude': 18.00396061};
+      case 'Uppsala Kungsgatan':
+        return {'latitude': 59.85953006, 'longitude': 17.64248414};
+      default:
+        return undefined;
+    }
+  };
+
+  // Load the html into Cheerio
+  var $ = cheerio.load(result, {decodeEntities: false});
+
+  // select the relevant JS nodes
+  var nodes = [];
+  $('.entry-content script').each(function () {
+    var rendered = $(this).html();
+    if (rendered.startsWith('\r\n\t')) {
+      // extract data table
+      rendered = rendered.substring(rendered.indexOf('arrayToDataTable('));
+      rendered = rendered.substring(18, rendered.indexOf(']);'));
+      rendered = rendered.replace('[', '').trim();
+      rendered = rendered.split("'").join('').trim(); // remove all [
+
+      if (rendered.length > 20) {
+        nodes.push(rendered);
+      }
+    }
+  });
+
+  var parameters = ['pm10', 'no2', 'pm25', 'o3']; // currently this order
+
+  // Iterate over different quantitites (PM10, NO2, PM25, O3)
+  nodes.forEach((node, nodeIndex) => {
+    // Parse data table
+    var rows = node.split('],[');
+    var legend = rows.shift().split(',');
+
+    // Iterate over the last 24 hours of measurements
+    rows.forEach((row, index) => {
+      row = row.split(']').join('').trim().split(',');
+
+      // First column contains the hour of the recordings
+      var date = moment(row[0], 'HH:mm').date(moment().date());
```

Comment:
Just did a quick spot check and time doesn't seem to be matching up. The adapter is giving me 

```json
{"date":{"utc":"2017-01-19T00:00:00.000Z","local":"2017-01-19T01:00:00+01:00"},"parameter":"no2","location":"Hornsgatan","value":43.76,"unit":"µg/m³","city":"Stockholm","attribution":[{"name":"SLB","url":"http://slb.nu/slbanalys/luften-idag/"}],"averagingPeriod":{"value":1,"unit":"hours"},"coordinates":{"latitude":59.31713214,"longitude":18.04878744},"country":"SE","sourceName":"Sweden","sourceType":"government","mobile":false}
```

but the page is showing that value at 19:00. Noticed this for other measurements as well.